src/components/radio-button/radiobutton.test.js

```diff
@@ -130,235 +87,247 @@ context("Testing RadioButton component", () => {
         radiobuttonRole().should(assertion);
       }
     );
-  });
 
-  it.each(testData)(
-    "should render Radiobutton component with %s as a label",
-    (label) => {
-      CypressMountWithProviders(<RadiobuttonComponent label={label} />);
+    it.each(testData)(
+      "should render Radiobutton component with %s as a label",
```

Comment:
As above with the suggested `RadioButton...` in line with the rest of the naming convention, although I can see that it was already like this when you did the refactor. I still think it should be changed to keep it in line with the naming convention throughout the rest of the file and other components.